nautobot/core/tests/test_filters.py

```diff
@@ -741,21 +781,15 @@ def setUpTestData(cls):
         )
         dcim_models.Platform.objects.bulk_create(platforms)
 
-        cls.regions = dcim_models.Region.objects.filter(sites__isnull=False)[:3]
-
-        cls.sites = (
-            dcim_models.Site.objects.filter(region=cls.regions[0]).first(),
-            dcim_models.Site.objects.filter(region=cls.regions[1]).first(),
-            dcim_models.Site.objects.filter(region=cls.regions[2]).first(),
-        )
-        cls.sites[0].asn = 65001
-        cls.sites[1].asn = 65101
-        cls.sites[2].asn = 65201
+        cls.locations = dcim_models.Location.objects.filter(parent__isnull=False)[:3]
```

Comment:
Probably should also restrict by location-type or otherwise ensure that the selected locations are ones that actually permit Racks? (Since we're using `bulk_create` for the Racks below it's bypassing any validation that we have there.)